firstvoices/backend/models/import_jobs.py

```diff
@@ -121,3 +122,18 @@ class Meta:
         on_delete=models.SET_NULL,
         related_name="import_job_failed_rows_csv_set",
     )
+
+    def _delete_report(self):
+        import_job_report = self.validation_report
+        if import_job_report:
+            import_job_report.delete()
+
+    def delete(self, using=None, keep_parents=False):
+        """
+        Does not allow deleting on an instance if the job has been completed, i.e. status="completed".
+        """
+        if self.status == JobStatus.COMPLETE:
```

Comment:
I think need to prevent deletion in some more cases as well:

- status = STARTED or ACCEPTED, since the task will still run if we delete the job, and we'll have confusing results
- validation_status = STARTED, because the running task will re-save the job and the validation report. In most cases validation_status = ACCEPTED would be ok, since when the task runs it gets the ImportJob early on and would fail, but with async things it is still possible that in between us checking the status and deleting the job, the task could start and get past that check. SO, we should also block deleting a job if validation_status is ACCEPTED